packages/dd-trace/src/plugins/util/user-provided-git.js

```diff
@@ -39,6 +41,35 @@ function filterSensitiveInfoFromRepository (repositoryUrl) {
   }
 }
 
+function validateGitRepositoryUrl (repoUrl) {
+  return /((http|git|ssh|http(s)|file|\/?)|(git@[\w.]+))(:(\/\/)?)([\w.@:/\-~]+)(\.git)(\/)?/.test(repoUrl)
```

Comment:
why is a group the `s` of `http(s)`?
and this regex accepts things like `/://aaa.git` is this expected?